incubator/binding-openapi/src/test/java/io/aklivity/zilla/runtime/binding/openapi/internal/config/OpenapiOptionsConfigAdapterTest.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.binding.openapi.internal.config;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Arrays.asList;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import jakarta.json.bind.Jsonb;
+import jakarta.json.bind.JsonbBuilder;
+import jakarta.json.bind.JsonbConfig;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+import io.aklivity.zilla.runtime.binding.openapi.config.OpenapiConfig;
+import io.aklivity.zilla.runtime.binding.openapi.config.OpenapiOptionsConfig;
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.OpenApi;
+import io.aklivity.zilla.runtime.binding.openapi.internal.model.PathItem;
+import io.aklivity.zilla.runtime.engine.config.ConfigAdapterContext;
+import io.aklivity.zilla.runtime.engine.config.OptionsConfigAdapterSpi;
+import io.aklivity.zilla.runtime.engine.internal.config.OptionsAdapter;
+
+public class OpenapiOptionsConfigAdapterTest
+{
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule();
+    @Mock
+    private ConfigAdapterContext context;
+
+    private OptionsAdapter adapter;
+
+    private Jsonb jsonb;
+
+
+    @Before
+    public void initJson() throws IOException
+    {
+        String content = null;
+        try (InputStream resource = OpenapiOptionsConfigAdapterTest.class
```

Comment:
I think it might be helpful to define an empty `OpenapiSpecs` class in `binding-openapi.spec` under `io.aklivity.specs.binding.openapi` package and make sure the `petstore.yaml` file is included as a resource in the `binding-openapi.spec` jar.

Then this can become `OpenapiSpecs.getResourceAsStream("config/openapi/petstore.yaml")` instead because the openapi specs jar is a test dependency and therefore on the test classpath for this openapi binding implementation project.